.github/workflows/python.yaml

```diff
@@ -137,7 +137,7 @@ jobs:
             usage_rate: "1.0"
           - name: source-pendo
             type: capture
-            version: v1
+            version: v2
```

Comment:
Any reason to do a version bump here? I wouldn't think we'd need to based on these changes the current usage of the connector.

Reply:
The main reason I did a version bump was because the endpoint config has `startDate` as a new required field. I agree that with how many tasks currently exist for this connector, we could keep it as `v1`. I'll undo the version bump.